kbase-extension/static/kbase/js/widgets/appWidgets2/validators/constants.js

```diff
@@ -11,7 +12,33 @@ define([], () => {
         ERROR: 'error',
     };
 
+    // error message ids. these are controlled, messages themselves are not,
+    // and should be user readable
+    const MESSAGE_IDS = {
+        INFINITE: 'infinite-value',
```

Comment:
Just making sure these are normalized. I'm usually not in favor of "someday we'll do something with this!" coding, but they're already sprinkled throughout the codebase.